cmd/otel-allocator/internal/collector/collector.go

```diff
@@ -20,7 +20,8 @@ import (
 )
 
 const (
-	defaultMinUpdateInterval = time.Second * 5
+	defaultMinUpdateInterval                 = time.Second * 5
+	defaultGracePeriodBeforeSkipBadCollector = time.Second * 30
```

Comment:
That's a wordy name. How about `defaultNotReadyGracePeriod`, and add a comment explaining what this is?

Reply:
Changed! Appreciate the suggestion.